src/core/notifier/topic/topics.go

```diff
@@ -0,0 +1,56 @@
+package topic
+
+import (
+	"github.com/goharbor/harbor/src/common"
+	"github.com/goharbor/harbor/src/common/utils/log"
+	"github.com/goharbor/harbor/src/core/notifier"
+	"github.com/goharbor/harbor/src/core/notifier/handler/webhook"
+)
+
+// Define global topic names
+const (
+	// ScanAllPolicyTopic is for notifying the change of scanning all policy.
+	ScanAllPolicyTopic = common.ScanAllPolicy
```

Comment:
Could you remove it, `ScanAllPolicyTopic ` is no longer needed

Reply:
> Could you remove it, `ScanAllPolicyTopic ` is no longer needed

OK